templates/blog-post.html

```diff
@@ -2,7 +2,16 @@
 {% block title %}{{ this.title }}{% endblock %}
 {% block header %}{{ this.parent.title }}{% endblock %}
 {% block body %}
-  <h2 class="mb-0">{{ this.title }}</h2>
-  <p class="meta text-muted mt-0">by {{ this.author }} on {{ this.pub_date|dateformat('full') }}
-  <div class="body">{{ this.body }}</div>
-{% endblock %}
+<h2 class="mb-0">{{ this.title }}</h2>
+<p class="meta text-muted mt-0">by {{ this.author }} on {{ this.pub_date|dateformat('full') }}</p>
+<div class="body">{{ this.body }}</div>
+<hr>
+<p class="meta mt-0 mb-0"><span class="text-muted">Categories: </span>
+  {% if this.categories|length %}
+    {% for category in this.categories %}
+    <a href="#">{{category}}</a>{{ "," if not loop.last}}
```

Comment:
Small style issue: could you ensure that there are spaces between the curly braces and the text inside on both sides for all your changes?

Reply:
okay

Reply:
@kgodey done!